### ISSUE TITLE
Balloons, Pyrolysis, Gas Tanks, Triglycerides

### ISSUE THREAD

**Issue by Phosgenick** [petrolpark/Destroy#305]
- Addition of ballons, made out of latex extracted from jungle logs being pressed in a basin ( maybe 10 mB per log). The balloons could store gases from the vat, just like the test tubes, but without the need to cool the contents. it could also be used to make an early game aging barrel.

- Reaction between phosgene and amines, producing ureas and isocyanates, which could be used to make foams and other polymers, like polyurethane.

- Urea synthesis from ammonia and phosgene in low temperature or ammonia and CO2 in high temperature and pressure. The urea could be used as a fertilizer or in the reaction with sodium hydroxide and sodium hypochlorite to yeald hydrazine, via the Hoffman degradation.

-Synthesis of amides by the reaction with the carboxylic acid at high temperature, and dehydration to nitriles. That would be a nice pathway to Nylon.

-Pyrolisis reactions on a furnace, special bubble cap or vat of some kind for useful reactions, such as pyrolisis of calcium acetate to yeald acetone, pyrolisis of chalk to yeald CO2 and CaO, hypochlorite decomposition to chloride and chlorate (for use as a energetic), etc...

-More electrophilic additions outside of nitration, such as all friedel-crafts reactions, addition of aldehydes or ketones with a acid catalyst, such as in the synthesis of DDT, which could make crops grow faster with the expence of killing bees, parrots and fishes close to the crops.

- Special gas tanks to store gases more eficiently at higher pressures.

- Chlorine having it's yellow-greenish color in the gas fase.

- Addition of parafine and kerosene to crude oil, as well as the cracking process, using  a silica catalys to produce more alkenes that just LPG alone.

- Synthesis of glycerol by the reaction of Porkshop or another animal fat with sodium hydroxide, converting the triglycerides to glycerol and oleic acid salts

- Some kind of pipebomb that uses gunpowder to make a weak explosive but that the player can throw and breake one block. Would be great to find oil.

-Some late game chemical weapon that throw really damaging chemicals on mobs. Maybe phosgene, lewisite or sulfur mustard based. 

-Vanadium pentoxide to, together with oxygen, substitute the magic oxidizer.

- Wolff-Kishner reduction.

-Nitro group reduction by iron or zinc in acid, to yeald amines, mainly for aromatic compounds.

That's only some of the sugestions. If you like, i will make more. Feel free to criticize.
Also, sorry for the bad english.

**Comment by petrolpark** [comment]
A lot of good stuff here! Quite a lot of it is actually already planned, so I guess that means I have the right idea of what people want? Thanks for the list, the English is fine.

**Comment by petrolpark** [comment]
> Addition of ballons, made out of latex extracted from jungle logs being pressed in a basin ( maybe 10 mB per log). The balloons could store gases from the vat, just like the test tubes, but without the need to cool the contents. it could also be used to make an early game aging barrel.

The concept you've proposed for balloons is one I considered before (and one I quite like), and I've seen the idea for balloons in general been floated around (no pun intended). I have a couple of neat ideas for tapping. The Aging Barrel is definitely getting made easier to craft. 

Likelihood: 🟠 reasonably likely

> Reaction between phosgene and amines, producing ureas and isocyanates, which could be used to make foams and other polymers, like polyurethane.

This is planned!

Likelihood: 🟢very likely

> Urea synthesis from ammonia and phosgene in low temperature or ammonia and CO2 in high temperature and pressure. The urea could be used as a fertilizer or in the reaction with sodium hydroxide and sodium hypochlorite to yield hydrazine, via the Hoffman degradation.

Interesting stuff. I'll research it more, but it sounds like it would fit in nicely (without adding too many new things). I believe there were plans to add urea in 1.0 anyway.

Likelihood: 🟢 quite likely

> Pyrolisis reactions on a furnace, special bubble cap or vat of some kind for useful reactions, such as pyrolisis of calcium acetate to yeald acetone, pyrolisis of chalk to yeald CO2 and CaO, hypochlorite decomposition to chloride and chlorate (for use as a energetic), etc...

There won't be any separate blocks/multiblocks that can only do certain reactions. These may be added as normal reactions, whose kinetic constants and side reactions require that they are done at relevant conditions (high pressure/temperature, anaerobic, etc).

Likelihood: 🔴 unlikely

> More electrophilic additions outside of nitration, such as all friedel-crafts reactions, addition of aldehydes or ketones with a acid catalyst, such as in the synthesis of DDT, which could make crops grow faster with the expence of killing bees, parrots and fishes close to the crops.

Lots of electrophilic additions are planned, and future versions (if not 1.0) will definitely have more aromatic reactions. DDT is planned, though not with that exact use.

Likelihood: 🟢 very likely

> Special gas tanks to store gases more eficiently at higher pressures.

Vats can be used to store gases with relatively low performance cost (as they do not do any computation when everything is still), and 1.0 will feature blocks with higher pressure capacities, so this isn't a high priority.

Likelihood: 🔴 unlikely for now

> Chlorine having it's yellow-greenish color in the gas fase.

Already in 1.0!

Likelihood: 🟢 definite

> Addition of parafine and kerosene to crude oil, as well as the cracking process, using a silica catalys to produce more alkenes that just LPG alone.

Additional products during oil refinement isn't super necessary, as they wouldn't serve much purpose within Destroy (other than just being distilled again). 1.0 will likely feature compatibility with things like TFMG, allowing you to further distil their crude oil fractions, if more complicated oil refining is desired. The original purpose of the Zeolite item was for cracking, and it may see that use return

Likelihood: 🔴 unlikely

> Synthesis of glycerol by the reaction of Porkshop or another animal fat with sodium hydroxide, converting the triglycerides to glycerol and oleic acid salts

This was planned at one point (though it would have come from Destroy's Cream). It will probably make a return in some form.

Likelihood: 🟠 likely

> Some kind of pipebomb that uses gunpowder to make a weak explosive but that the player can throw and breake one block. Would be great to find oil.

The way you search for Crude Oil is being made easier, meaning that this hopefully won't be as much of an issue. Even so, it is likely that Destroy will add in some weak early-game replacement for the TNT it makes more difficult to craft.

Likelihood: 🟠 not unlikely

> Vanadium pentoxide to, together with oxygen, substitute the magic oxidizer.

The magic oxidizer is obviously getting replaced. Vanadium probably won't be added *soon* as it wouldn't have many uses outside of that. As 1.0 will likely cover redox, there will be other catalysts you can use.

> Wolff-Kishner reduction

This is planned!

Likelihood: 🟢 definite

> Nitro group reduction by iron or zinc in acid, to yeald amines, mainly for aromatic compounds.

This will probably make it in.

Likelihood: 🟢 quite likely

**Comment by Phosgenick** [comment]
Nice, the majority is planned or likely! I really think these new additions
will make the game even more complex, but not in a hard way, just more
content to have fun.

 I have to say, I really like the idea of pyrolysis, even if it's not very
likely. The idea of getting energetic chemicals through heat could mean the
addition of more fuel sources that are stronger than the blaze cake,
meaning even higher temperatures than 327C could be reached in the bubble
cap, being able to distil from sulfuric acid to mercury. It's also a nice
way of getting access to chemicals in an easy fashion, but with a hard to
get setup ( considering a high price for the energetic fuel).

I just meant another reaction vessel because of the low uses for nickel,
maybe making some sort of special alloy to sustain high heat or pressure.

But i really have to say, even though there are lots of things still to
change or be added, i'm already quite addicted!

**Comment by petrolpark** [comment]
Agreed, Destroy's interplay is one of its strongest points, IMO. The urea suggestion is perfect, as without adding anything new it stitches together existing features and adds depth that way. That's the type of thing I always try to look for with new features.

Literal pyrolysis is already possible (and is already a thing with how you make PTFE). It's just a reaction with only one reactant and a high activation energy. You may be thinking of something different?

Nickel will be getting a large sink in 1.0.

**Comment by Phosgenick** [comment]
Oh yeah, i forgot about the tetrafluoroethene.  I meant by pyrolisis a way
of not just adding energy to the system, but heat itself. Maybe keeping
water having the high specific heat it has in the game and also other
substances too, but take more time to heat up, and eventually became vapors
(of course generating pressure, which have to be released). Maybe each
blaze burner having a Max temperature, like 50⁰C in normal Mode and 100⁰C
super heated (and even more with a new fuel source). But overal decomposing
substances based on the temperature of the vat.

**Comment by Phosgenick** [comment]
Sorry, meant vat, not basin

**Comment by Phosgenick** [comment]
2025-03-04_00-12-55_McPhos (1).zip
<https://drive.google.com/file/d/1k6vtRdqXYBVhN_d-r2tfqRSHJ7iwAEhQ/view?usp=drive_web>
